source/lib/defaults.js

```diff
@@ -2,17 +2,21 @@ export const notificationReasons = new Map([
 	['subscribed', 'You are watching the repository'],
 	['manual', 'You are subscribed to this thread'],
 	['author', 'You created this thread'],
-	['comment', 'New comment'],
-	['mention', 'You were mentioned'],
-	['team_mention', 'Your team was mentioned'],
+	['comment', 'You commented on this thread'],
+	['mention', 'New updates from thread'],
+	['team_mention', 'New updates from thread'],
 	['state_change', 'Thread status changed'],
-	['assign', 'You were assigned to the issue']
+	['assign', 'You were assigned to the thread'],
+	['security_alert', 'New security vulnerability found'],
+	['invitation', 'You accepted an invitation'],
+	['review_requested', 'PR Review Requested']
 ]);
 
 export const errorTitles = new Map([
 	['missing token', 'Missing access token, please create one and enter it in Options'],
 	['server error', 'You have to be connected to the Internet'],
 	['client error', 'Invalid token, enter a valid one'],
+	['network error', 'Failed to fetch Github Notification API'],
```

Comment:
Messages for `'network error'` and `'server error'` should be exchanged here.

And for server errors make it to something like "GitHub having issues serving requests"? Feel free to change it to something relevant.

Reply:
Agreed, will do. Thanks